extensions/scarb-cairo-run/tests/arguments.rs

```diff
@@ -335,3 +335,26 @@ fn cannot_set_gas_limit_for_package_with_disabled_gas_calculation() {
             error: gas calculation disabled for package `hello`, cannot define custom gas limit
         "#});
 }
+
+#[test]
+fn can_control_verbosity() {
```

Comment:
I like all tests added in this PR. Let's leave them even if we rewrite the underlying logic.